src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+logo.addEventListener('click', () => {
+  handleClick();
+});
+
+const handleClick = () => {
```

Comment:
This function `handleClick` is being referenced before it is declared. JavaScript has function hoisting, which means function declarations are moved to the top of their containing scope. However, function expressions, like this one, are not hoisted. As a result, you get a `ReferenceError` when trying to assign the function to an event listener before declaring it. To fix this, you should move the function declaration above where it's being used.